### DISCUSSION TITLE
Would use weblate to translate  po files?

### DISCUSSION THREAD

**Discussion by atzlinux** [hardinfo2/hardinfo2#43] [Ideas]
https://hosted.weblate.org/  is easy to use translate  po file, also has many open source projects use it.

Would hardinfo2 use weblate to translate  po files?

**Comment by hwspeedy** [comment]
It seams like a website that hosts the translations, so not the obvious choice for GNU Linux FOSS projects using git. Users of hardinfo2, that wants to translate runs Linux, loves tech and knows poedit and it has an established track record of stability.

**README.md for translators:**
https://github.com/hardinfo2/hardinfo2/tree/master/po

**Reply by roke-julian-lockhart** [reply]
> https://github.com/hardinfo2/hardinfo2/discussions/43#discussioncomment-9560831

@hwspeedy, weblate is FOSS, per https://github.com/WeblateOrg/weblate/tree/a59879d8dfebf354eda83e71945050c717b1b1d4. https://docs.weblate.org/en/latest/formats/gettext.html especially may be of interest.

**Reply by hwspeedy** [reply]
https://hosted.weblate.org/about/

Does not seam like FOSS to me and the idea of non git hosting is not my cup of tea.
Show an PoC showing what we gain, if you are keen on weblate.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/hardinfo2/hardinfo2/discussions/43#discussioncomment-10052861

@hwspeedy, [the Open Document Foundation considers it FOSS.](https://blog.documentfoundation.org/blog/2023/08/16/weblate-translation-tool-improvements-funded-by-tdf/#:~:text=Weblate%20is%20a%20free%20and%20open%20source%20software%20(FOSS)%20translation%20tool%2C%20with%20version%20control%20features.) Though, I agree that non-`git` hostage might be annoying for developers. I think its primary purpose it to make translation of software easily achievable by non-software developers - translators.
